### PR TITLE
issue: 1902084 Add deprecated message for Multi Packet RQ

### PR DESCRIPTION
**Pull request by igor-ivanov**
Signed-off-by: Igor Ivanov <igor.ivanov.va@gmail.com>

### CODE REVIEW

File: config/m4/verbs.m4 [hunk 1]
```diff
@@ -186,7 +186,7 @@ if test "x$vma_cv_verbs" == x2; then
 
     AC_MSG_CHECKING([for multi packet RQ support])
     AS_IF([test "x$have_mp_rq" == xyes -a "x$enable_exp_cq" == xyes -a "x$vma_cv_directverbs" == x2],
-	    [AC_DEFINE([HAVE_MP_RQ], 1, [MP_RQ QP supported])] [AC_MSG_RESULT([yes])],
+	    [AC_DEFINE([HAVE_MP_RQ], 1, [MP_RQ QP supported])] [AC_MSG_RESULT([yes (warning: this feature is deprecated and will be removed in a future release)])],
```
Comment (ilansmith) [inline config/m4/verbs.m4]: Is it **already** deprecated or will it be deprecated in future releases?

File: src/vma/sock/sock-redirect.cpp [hunk 2]
```diff
@@ -613,6 +613,9 @@ int vma_dump_fd_stats(int fd, int log_level)
 	return -1;
 }
 
+/* Multi Packet Receive Queue functionality is deprecated
+ * and not going to be supported in the future release
```
Comment (ilansmith) [inline src/vma/sock/sock-redirect.cpp]: \* and **is** not going to be supported in future release**s**

File: src/vma/sock/sock-redirect.cpp [hunk 3]
```diff
@@ -641,6 +644,9 @@ int vma_cyclic_buffer_read(int fd, struct vma_completion_cb_t *completion,
 #endif // HAVE_MP_RQ
 }
 
+/* Multi Packet Receive Queue functionality is deprecated
+ * and not going to be supported in the future release
+ */
```
Comment (ilansmith) [inline src/vma/sock/sock-redirect.cpp]: Same as above.

File: config/m4/verbs.m4 (hunk 1 above)
Comment (igor-ivanov) [reply config/m4/verbs.m4]: It is deprecated but exist for 8.9.x and will be removed after

File: src/vma/sock/sock-redirect.cpp (hunk 2 above)
Comment (igor-ivanov) [reply src/vma/sock/sock-redirect.cpp]: updated

File: src/vma/sock/sock-redirect.cpp (hunk 3 above)
Comment (igor-ivanov) [reply src/vma/sock/sock-redirect.cpp]: updated

**Pull request merged**